en/docs/data-transformation/guides/hl7v2-fhir.md

```diff
@@ -0,0 +1,364 @@
+## Introduction
+The hl7v2 to fhirr4 library is a Ballerina module designed to facilitate the conversion of healthcare data from the HL7 version 2 (HL7v2) standard to the FHIR R4 standard. In the healthcare industry, interoperability and data exchange between different systems are critical for ensuring accurate and efficient patient care. This library provides developers with a straightforward way to transform HL7v2 messages into FHIR R4 resources, enabling seamless integration with modern healthcare systems that utilize the FHIR standard.
+
+## Mapping ADT_A01 Message to FHIR r4
+In hl7v2, there are Messages, Segments and Data types that we can map to FHIR. In this example let's see how we can map ADT_A01.PID segment to FHIR Patient. 
+
```

Comment:
Let's add the note that we have two implementation flavours

{!includes/bal-mi-note.md!}

Reply:
This is fixed.